examples/clustering/plot_dtw_boss.py

```diff
@@ -0,0 +1,104 @@
+"""
+=========================================
+Time Series Clustering with DTW and BOSS
+=========================================
+
+This example shows the differences between various metrics
+related to time series clustering. Besides the Euclidean distance,
+:func:`pyts.metrics.dtw` and :func:`pyts.metrics.boss` are considered to
+analyze the :func:`pyts.datasets.make_cylinder_bell_funnel` dataset.
+In contrast to reference [1], DTW-based clustering shows the best results
+here - even when repeated over multiple seeds. The reason for this could be
```

Comment:
I don't think that you repeat the experiments with different random seeds now. Even if you tried it out, I'm not sure that it should be mentioned in the documentation of this example if it does not illustrate it.